<ui version="4.0" >
 <class>HistoryWidget</class>
 <widget class="QWidget" name="HistoryWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>155</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Description of the current step:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="description" >
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="undoRedoEnabled" >
      <bool>false</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonFirst" >
       <property name="toolTip" >
        <string>First step</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack" >
       <property name="toolTip" >
        <string>One step back</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="editStep" />
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSteps" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext" >
       <property name="toolTip" >
        <string>One step forward</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonLast" >
       <property name="toolTip" >
        <string>Last step</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
